maisim/maisim.Game/maisimGame.cs

```diff
@@ -52,10 +64,21 @@ protected override void LoadComplete()
                         }
                     }
                 },
+                overlayOffsetContainer = new Container

```

Comment:
These doesn't seem to be used for the moment (no?)

Reply:
Yep I add it because I need it when support for more platform but it's a very long future so just remove it for now.